modules/hello-world/pages/platform-help.adoc

```diff
@@ -0,0 +1,172 @@
+= Setting Up Couchbase PHP SDK with VSCode
+:description: pass:q[Discover how to get up and running developing applications with the Couchbase PHP SDK 3.0+ using `Visual Studio Code`.]
```

Comment:
Are we adding this to the 3.x docs as well, then?

Reply:
oops